VM/rel2vm_params.py

```diff
@@ -871,19 +897,27 @@ def load_args(logger: Logger = qclogging.get_basic_logger()):
     arg(
         "--vm-topo",
         help="topo_type parameter for velocity model generation",
+        choices=[x.value for x in constants.VelocityModelTopographyType],
         default="BULLDOZED",
     )
     arg(
-        "--no_optimise",
+        "--no-optimise",
         help="Don't try and optimise the vm if it is off shore. Removes dependency on having GMT coastline data",
         action="store_true",
+        default=False,
```

Comment:
If we are refactoring this one, it would be a good idea to have a dest called "optimisation" (or "do_optimisation" etc) rather than having to work out inverted logic.

Reply:
Please elaborate. Not sure what you mean by "dest"

Reply:
The add_argument has a parameter called "dest", which becomes the name of the argument when retrieved from the args object. e.g. 
add_argument("--no-optimise", action="store_false", dest="optimisation")
args.optimisation then has the appropriate value, instead of args.no_optimise

Reply:
Thanks